docs/lambda-selector/lambda-attributes-selector.asciidoc

```diff
@@ -106,7 +127,9 @@ window.addEventListener("DOMContentLoaded", async () => {
 });
 </script>
 
-<div role="lambda-selector">
+<p id="fallback-extension-arn-selector-section">Pick the right ARN from <a target="_bkank" href="RELEASE_LINK">this release table for the APM Lambda Extension Layer</a>.</p>
+<p id="fallback-agent-arn-selector-section">In addition, pick the right ARN from <a target="_bkank" href="RELEASE_LINK">this release table for the APM Agent Layer</a>.</p>
```

Comment:
```suggestion
<p id="fallback-extension-arn-selector-section">Pick the right ARN from <a target="_blank" href="RELEASE_LINK">this release table for the APM Lambda Extension Layer</a>.</p>
<p id="fallback-agent-arn-selector-section">In addition, pick the right ARN from <a target="_blank" href="RELEASE_LINK">this release table for the APM Agent Layer</a>.</p>
```

Were these intended to be `target="_blank"`?

Reply:
Fixed on `main` in d0f03e72b9e5d467dd375a60fc8ef98a1d34b29c.